test/functional/feature_assumeutxo.py

```diff
@@ -644,7 +661,8 @@ def check_tx_counts(final: bool) -> None:
         for i in (0, 1):
             n = self.nodes[i]
             self.log.info(f"Restarting node {i} to ensure (Check|Load)BlockIndex passes")
-            self.restart_node(i, extra_args=self.extra_args[i])
+            with self.assert_disk_cleanup(n, True if i == 1 else False):
```

Comment:
Nit: I think you can just pass in `i==1` here, without the `if` `else` (and similarly below).

Reply:
Indeed, nice.